docs/spec/v1beta3/providers.md

```diff
@@ -778,6 +778,75 @@ stringData:
     attr2-name: attr2-value
 ```
 
+###### Google Pub/Sub with Workload Identity Example
+
+To configure a Provider for Google Pub/Sub authenticating with Workload Identity,
+create a Kubernetes ServiceAccount with the appropriate annotations, and a
+`googlepubsub` Provider with the ServiceAccount reference. This method eliminates
+the need for JSON credentials and enables multi-tenant authentication.
+
+**Single tenant approach**
+
+This approach uses the notification-controller service account for setting up
+authentication.
+
+- In the default installation, the notification-controller service account is
+  located in the `flux-system` namespace with name `notification-controller`.
+
+- Configure workload identity with notification-controller as described in the
+  docs [here](/flux/installation/configuration/workload-identity/).
```

Comment:
```suggestion
  docs [here](/flux/integrations/gcp/).
```

Reply:
Let's link also directly to https://fluxcd.io/flux/integrations/gcp/#for-google-cloud-pubsub, which is the specific section for Pub/Sub